### DISCUSSION TITLE
disappeared string

### DISCUSSION THREAD

**Discussion by tomkolp** [WeblateOrg/weblate#5894] [Q&A] [answered]
Today, after clicking "Download translation files as ZIP file", some incomprehensible errors appeared.
[Sentry](https://sentry.io/share/issue/3a78ba09deaa402f816d5de387920af1/)
It is also impossible to make a commit.
4.6 — [25929f7902c3401abf145c13aeff51b982da1032](https://github.com/WeblateOrg/weblate/commits/25929f7902c3401abf145c13aeff51b982da1032)
![image](https://user-images.githubusercontent.com/39633242/115505361-9e915800-a279-11eb-9aa3-43ee350e70b3.png)
![image](https://user-images.githubusercontent.com/39633242/115506374-12803000-a27b-11eb-932e-3dbdc1f02fb0.png)

**Comment by nijel** [comment, marked as answer]
Weblate could not find the string in the file when committing pending changes. This either indicates a bug in the file format handler or the file on the disk got out of sync with the content in the database.

**Reply by tomkolp** [reply]
What can I do to avoid losing 12,000 changes from the last 4 days?

**Reply by nijel** [reply]
You can download content in converted format and you will get everything what is in the databse.
